client/app/components/onboardingGuide/GuideCoverPhotoPage.js

```diff
@@ -29,18 +30,25 @@ const GuideCoverPhotoPage = (props) => {
       }),
       div({
         className: css.infoTextContent,
-        dangerouslySetInnerHTML: { __html: t('advice') }, // eslint-disable-line react/no-danger
-      }),
+      }, [
+        t('web.admin.onboarding.guide.cover_photo.advice.content1',
+          { link: a({
+            href: 'http://support.sharetribe.com/knowledgebase/articles/744438',
+            target: '_blank',
+            alt: t('web.admin.onboarding.guide.cover_photo.advice.alt'),
+          }, t('web.admin.onboarding.guide.cover_photo.advice.link')) }),
+        br(),
+        t('web.admin.onboarding.guide.cover_photo.advice.content2', { width: 1920, height: 450 }),
```

Comment:
Maybe I was a bit too implicit here. I'm not after what those dimensions mean here in the review, but they should be clear from the code, e.g. `const COVER_PHOTO_DIMENSIONS = { width: 1920, height: 450 };`.